FiveCalls/FiveCalls/YourImpact.swift

```diff
@@ -49,54 +49,49 @@ struct YourImpact: View {
             
     var body: some View {
         NavigationStack {
-            VStack(spacing: 0) {
-                VStack(alignment: .leading, spacing: 15) {
-                    Text(weeklyStreekMessage)
+            List {
+                VStack(alignment: .leading, spacing: 16) {
+                    Text(weeklyStreakMessage)
```

Comment:
Does including the streak message in the list have an side effects or is the layout the same?